scripts/make_fake_pm_data.js

```diff
@@ -0,0 +1,134 @@
+const async = require("async");
+const fs = require("fs");
+const _ = require("underscore");
+const json2csv = require("json2csv");
+const argv = process.argv.slice(2);
+const BUILDINGS = require("./" + argv[0]);
+
+function randomNumber(min, max) {
+    return Math.floor(Math.random() * (max - min + 1) + min);
+}
+
+function randomNumberOfStandardDeviations(min, max) {
+    return ((Math.random() * (max - min + 1) + min))
+}
+
+function shiftCoordinates(coordinate, shiftValue) {
+    return (coordinate - shiftValue);
```

Comment:
Not sure this function is yet carrying its weight... it's not quite a whole concept over and above subtraction.